### ISSUE TITLE
Authentication failed with valid credentials

### ISSUE THREAD

**Issue by rbrdevs** [sentinel-hub/sentinelhub-qgis-plugin#39]
Hi,
I can't log in with the plugin for the past few days. 
Whenever I try, I always get the following error message in QGIS: 
`"CRITICAL    SentinelHub : Authentication failed, check your credentials"`
The credentials used are correct since I can connect successfully with Python or Postman. Recently, in a personal project, I've been forzed to update in  the SentinelHub Python library to version 3.9.2 or higher because they must have changed something in the authentication part of the SentinelHub API. This may be the reason for the error.

**Comment by Adrok24** [comment]
Hi, which version do you have? did you update to the last version of the plugin? Which endpoint are you using?

**Comment by rbrdevs** [comment]
Hi, 
I have plugin version 2.0.2 installed via the QGis plugin manager. 
As "Service URL" I am using 'https://services.sentinel-hub.com' 
I have tried with QGIS versions 3.28.12 and 3.34.0, getting the same error in both

**Comment by rbrdevs** [comment]
I just tested again, but instead of using the current credentials, I created new ones. It seems to work now with these new credentials. 
For some reason, the OAuth client authentication doesn't work with the ones I usually use. 
As I mentioned before, those previous credentials work fine, but apparently, it's necessary to update the SentinelHub Python library to version >=3.9.2 for it to function correctly
